app/src/main/java/com/codesoom/assignment/exception/ProductNotFoundException.java

```diff
@@ -0,0 +1,13 @@
+package com.codesoom.assignment.exception;
+
+/**
+ * 장난감을 찾지 못했을 때 발생하는 예외
+ */
```

Comment:
이렇게 해봅시다. 시간 여유가 있다면 유명한 예외 클래스들의 JavaDoc도 찾아서 읽어보세요.

```suggestion
/**
 * 상품을 찾지 못했을 때 던집니다.
 */
```